src/Microsoft.AspNetCore/CertificateLoader.cs

```diff
@@ -163,7 +163,7 @@ public override X509Certificate2 Load()
 
                 if (error != null)
                 {
-                    throw error;
+                    throw new CertificateConfigurationException($"Unable to load certificate from file '{Path}': {error.Message}");
```

Comment:
`Unable to load certificate from file '{Path}'. Error details: '{error.Message}'.`